client/src/includes/a11y-result.ts

```diff
@@ -0,0 +1,136 @@
+import {
```

Comment:
Of all four functions here, only `renderA11YResults` is new. The other ones were previously defined inside `usebar.ts`.

Reply:
👍 
Just for consistency with other names, should we use lowercase `y`, i.e. `renderA11yResults`?